test/balance_edges.jl

```diff
@@ -0,0 +1,35 @@
+using LightGraphs
+using Random
+using DataStructures
+
+""" Refer to test/graph.jl to see the test graph being loaded
+"""
+
```

Comment:
Could we also write a tiny test for the `kruskal_mst` method if it's not too much trouble? 😄

Reply:
of course!!! thanks for keeping me honest

Reply:
done and passed!